README.md

```diff
@@ -68,6 +68,28 @@ steps:
               secret-id: 'arn:aws:secretsmanager:ap-southeast-2:1234567:secret:my-global-file-secret'
 ```
 
+## For Secrets in Another Region
+
+In order to request a secret from another region from your agents, a region parameter must be provided.
+This is independent of the secret ARN, which may confusingly include a region.
+
+```yml
+steps:
+  - commands: 'echo \$MY_LOCAL_SECRET and \$MY_GLOBAL_SECRET'
+    plugins:
+      - seek-oss/aws-sm#v0.0.5:
+          env:
+            - secret-id: 'arn:aws:secretsmanager:us-west-2:1234567:secret:my-global-secret'
+              env-name: MY_GLOBAL_SECRET
+              region: us-west-2
```

Comment:
Couldn't you infer the region from the ARN the same way we do in `secretsmanager.py`?

Reply:
That's a much better idea, and also addresses Darren's point. Will update.